pinkopy/base_session.py

```diff
@@ -1,11 +1,11 @@
 from base64 import b64encode
-from datetime import timedelta
```

Comment:
Some of these imports are resolved on dev, so they should be easy resolutions. Just take what is on develop in these cases.